autofaiss/external/quantize.py

```diff
@@ -146,6 +160,15 @@ def build_index(
     if ids_path is not None:
         ids_path = make_path_absolute(ids_path)
 
+    if nb_indices_to_keep < 1:
+        logger.error("Please specify nb_indices_to_keep an integer value larger or equal to 1")
+    if nb_indices_to_keep > 1:
+        if distributed is None:
+            logger.error('nb_indices_to_keep can only be larger than 1 when distributed is "pyspark"')
+            return None, None
+        current_memory_available = (
```

Comment:
I don't think we should implement it that way
That makes this variable contain something that is incorrect (even if we increase the number of indices that doesn't increase the size of the ram) and can break things down the line
The memory in autofaiss is used to:
* put in memory embeddings to train
* put in memory embeddings to add
* put in memory indices

the 2 first points are not affected by this multi indices change and shouldn't be impacted

only the last point needs to be impacted
As a consequence, I think the best way to go is to pass the nb_indices_to_keep param to the function that computes the memory estimation, to decrease the size needed to store indices in memory